contents/blog/the-posthog-array.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2023-02-24
+title: "What’s new in PostHog?"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+tags:
+  - Product updates
+  - Release notes
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Interested about what new features, fixes and updates have just launched on PostHog Cloud? Every Friday we update this article with a look back over the last seven days, so you can get a fresh look at what’s launched and launching right now. We also feature notable additions to the blog and docs. 
```

Comment:
```suggestion
Interested about what new features, fixes and updates have just launched on PostHog? Every Friday we update this page with a look back over the last seven days, so you can get a fresh look at what’s launched and launching right now. We also feature notable additions to the blog and docs. 
```

It's just PostHog now, right, rather than Cloud specifically?

Reply:
```suggestion
Interested about what new features, fixes and updates have just launched on PostHog Cloud? Every Friday we update this page with a look back over the last seven days, so you can get a fresh look at what’s launched and launching right now. We also feature notable additions to the blog and docs. 
```

Reply:
Well, we still have the Open Source Docker Compose version, so...